examples/101.03-bundle.json

```diff
@@ -0,0 +1,109 @@
+{
+  "credentials": {
+    "hostkey": {
+      "env": "HOST_KEY",
+      "path": "/etc/hostkey.txt"
+    }
+  },
+  "custom": {
+    "com.example.backup-preferences": {
+      "frequency": "daily"
+    },
+    "com.example.duffle-bag": {
+      "icon": "https://example.com/icon.png",
+      "iconType": "PNG"
+    },
+    "dependencies": [
+      {
+        "requires": {
+          "bundle": "azure/mysql",
+          "version": {
+            "prereleases": "true",
+            "range": "5.7.x"
+          }
+        }
+      }
+    ]
+  },
+  "definitions": {
+    "http_port": {
+      "default": 80,
+      "maximum": 10240,
+      "minimum": 10,
+      "type": "integer"
+    },
+    "port": {
+      "maximum": 65535,
+      "minimum": 1024,
+      "type": "integer"
+    },
+    "string": {
+      "type": "string"
+    },
+    "x509Certificate": {
+      "contentEncoding": "base64",
+      "contentMediaType": "application/x-x509-user-cert",
+      "type": "string",
+      "writeOnly": true
+    }
+  },
+  "description": "An example 'thin' helloworld Cloud-Native Application Bundle",
+  "images": {
+    "my-microservice": {
+      "contentDigest": "sha256:aaaaaaaaaaaa...",
+      "description": "my microservice",
+      "image": "technosophos/microservice:1.2.3"
+    }
+  },
+  "invocationImages": [
+    {
+      "contentDigest": "sha256:aaaaaaa...",
+      "image": "technosophos/helloworld:0.1.0",
+      "imageType": "docker"
+    }
+  ],
+  "maintainers": [
+    {
+      "email": "matt.butcher@microsoft.com",
+      "name": "Matt Butcher",
+      "url": "https://example.com"
+    }
+  ],
+  "name": "helloworld",
+  "outputs": {
+    "fields": {
+      "clientCert": {
+        "definition": "x509Certificate",
+        "path": "/cnab/app/outputs/clientCert"
+      },
+      "hostName": {
+        "applyTo": [
+          "install"
+        ],
+        "definition": "string",
+        "description": "the hostname produced installing the bundle",
+        "path": "/cnab/app/outputs/hostname"
+      },
+      "port": {
+        "definition": "port",
+        "path": "/cnab/app/outputs/port"
+      }
+    }
+  },
+  "parameters": {
+    "fields": {
+      "backend_port": {
+        "definition": "http_port",
+        "description": "The port that the back-end will listen on",
+        "destination": {
+          "env": "BACKEND_PORT"
+        }
+      }
+    }
+  },
+  "requiredExtensions": {
+    "dependencies": "cnab.io/specs/depenencies.schema.json"
```

Comment:
Needs to be an array.

Reply:
Doh, forgot to save the example when I pushed the rest;. Just fixed that.